src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+> This negativity is the interaction of the `mechanical object` with another
+> `mechanical object` – whereby a `mechanical object` has both an identical
+> determinateness to another `mechanical object`, whilst being indifferent to
+> it.
+
+The negativity of the contradiction, then, logically unfolds as the positing of
+their identical determinateness. At this point, one might legitimately ask: why
+is the first positing that of their identity and not that of their indifference
+to each other? The answer must be: because it is their identity that binds them
+into a relation with one another. It is because they have an identical
+determinateness that there is a negative _unity_ in the first place, and it is
+because of their identical determinateness that an interaction can take place.
+Thus, the identical relation is posited.
+
+The logical result of this positing is that the identical determinateness of the
+`mechanical objects` is itself determined. In other words, the logical structure
+of that identity is posited and in being posited is hardened (determined) into
+an actual moment between `mechanical objects`. Their identity is no longer just
+one side of a contradiction, but it has its day in the Sun, it is determined as
+a moment of what it is for the `mechanical object` to be in a formal mechanical
+relation with another `mechanical object`.
+
+Accordingly, because their identity is determined, their relation takes the form
+of a `universal` – there is nothing other than `mechanical objects` and these
+`mechanical objects` relate to each other as identical.[^1]:
+[first-paragraph](hegel/reference/mechanical-object/first-paragraph) Hegel calls
+this `communication` and, crucially, adds that it “does not involve transition
+into an opposite” (Hegel 1991, 716). The `communication` between
+`mechanical objects` is a movement of identical determinations, and so strictly
+speaking, there is no change from one thing to something different. It is quite
+simply the identical determinateness that exists between `mechanical objects` –
+a kind of ether of determinateness that connects all objects together but does
+not affect them beyond connecting them.
+
+### Communication Between Objects
+
+Hegel then embarks on a remark that helpfully unpacks what he has in mind with
+the logical determination of `communication`. He writes:
+
+> _Spiritual communication_, which moreover takes place in that element which is
+> the universal in the form of universality, is explicitly an _ideal_ relation
+> in which _a determinateness continues_ itself from one person into another
+> unimpaired, and universalizes itself without any alteration whatever-as a
+> scent freely spreads in the unresisting atmosphere. But even in communication
+> between material objects, their determinateness _spreads_, so to speak, in a
+> similarly ideal manner; personality is an infinitely more intense
+> _impenetrability_ [*Harte*] than objects possess (Hegel 1991, 716). The above
```

Comment:
Looks like part of your own text got tangled into the quote. Seperate it out into its own paragraph. Make sure to have an empty line between the quote and the new paragraph.